layouts/shortcodes/cncf-landscape.html

```diff
@@ -56,8 +56,15 @@
 });
 </script>
 {{- end -}}
+{{ if in (.Site.BaseURL | string) "https://kubernetes.io/partners" }}
```

Comment:
You shouldn't need to match on the URL where the shortcode is used. Instead, add `data-*` elements to the page where you're using it.

Reply:
Maybe the shortcode could take a new parameter, though, `category` (eg `kubernetes-training-partner`).

Reply:
What are the possible categories?